docs/source/general/setup.rst

```diff
@@ -128,8 +128,64 @@ In case you are using `openwisp-radius <https://github.com/openwisp/openwisp-rad
     ./manage.py migrate --fake openwisp_radius 0001_initial_freeradius
     ./manage.py migrate
 
-Installing for development
---------------------------
+
+Automating management commands
```

Comment:
this section should not be moved, it was good to keep it below, undo this change please